pkg/virt-config/virt-config.go

```diff
@@ -155,6 +155,11 @@ func (c *ClusterConfig) GetMemoryOvercommit() int {
 }
 
 func (c *ClusterConfig) GetEmulatedMachines(arch string) []string {
+	oldEmulatedMachines := c.GetConfig().EmulatedMachines
```

Comment:
@acardace @fossedihelm  IIUC we can't recognize whether the returned value is a default that came from defaultClusterConfig() or a user-defined value. EmulatedMachines will always carry some value.

Reply:
It means that if the user already opted-in for `ArchitectureConfiguration` with this change we will return cluster default